sentry_sdk/scope.py

```diff
@@ -559,6 +589,119 @@ def add_breadcrumb(self, crumb=None, hint=None, **kwargs):
         while len(self._breadcrumbs) > max_breadcrumbs:
             self._breadcrumbs.popleft()
 
+    def capture_event(self, event, hint=None, client=None, scope=None, **scope_kwargs):
+        # type: (Event, Optional[Hint], Optional[sentry_sdk.Client], Optional[Scope], Any) -> Optional[str]
+        """
+        Captures an event.
+
+        Merges given scope data and calls :py:meth:`sentry_sdk.Client.capture_event`.
+
+        :param event: A ready-made event that can be directly sent to Sentry.
+
+        :param hint: Contains metadata about the event that can be read from `before_send`, such as the original exception object or a HTTP request object.
+
+        :param client: The client to use for sending the event to Sentry.
+
+        :param scope: An optional :py:class:`sentry_sdk.Scope` to apply to events.
+            The `scope` and `scope_kwargs` parameters are mutually exclusive.
+
+        :param scope_kwargs: Optional data to apply to event.
+            For supported `**scope_kwargs` see :py:meth:`sentry_sdk.Scope.update_from_kwargs`.
+            The `scope` and `scope_kwargs` parameters are mutually exclusive.
+
+        :returns: An `event_id` if the SDK decided to send the event (see :py:meth:`sentry_sdk.Client.capture_event`).
+        """
+        if client is None:
+            return None
+
+        if scope_kwargs is not None:
```

Comment:
I think you still want to call `_merge_scopes` even if `scope_kwargs` is `None` but `scope` is not. This is a 3-way merge.

Reply:
Right. Checked also with master and there is no condition there, so I removed it. (_merge_scopes will raise an error when both (scope_kwargs and scope) are set)